Reply/app/src/main/res/values-night/themes.xml

```diff
@@ -19,24 +19,40 @@
 
     <style name="Theme.Reply.Dark" parent="Theme.Reply">
         <!--Color-->
-        <item name="colorPrimary">@color/reply_blue_200</item>
-        <item name="colorPrimaryVariant">@color/reply_blue_300</item>
-        <item name="colorSecondary">@color/reply_orange_300</item>
-        <item name="colorSecondaryVariant">@color/reply_orange_300</item>
-
-        <item name="android:colorBackground">@color/reply_black_900</item>
-        <item name="colorSurface">@color/reply_black_800</item>
-        <item name="colorPrimarySurfaceVariant">?attr/colorSurface</item>
```

Comment:
This is a custom Reply theme attr that's used for the bottom nav drawer [here](https://github.com/material-components/material-components-android-examples/blob/86b30c95c6984ba95f38cab77ea52dc74d3348cd/Reply/app/src/main/java/com/materialstudies/reply/ui/nav/BottomNavDrawerFragment.kt#L98). 

Could you remove the attr from `res/values/attrs.xml` and replace its usage in BottomNavDrawerFragment with `colorPrimaryContainer` for now?